uavcan/driver/__init__.py

```diff
@@ -28,10 +29,13 @@ def make_driver(device_name, **kwargs):
     """
     windows_com_port = device_name.replace('\\', '').replace('.', '').lower().startswith('com')
     unix_tty = device_name.startswith('/dev/')
+    pythoncan = device_name.startswith('pythoncan')
 
     if windows_com_port or unix_tty:
         return SLCAN(device_name, **kwargs)
     elif SocketCAN is not None:
         return SocketCAN(device_name, **kwargs)
+    elif pythoncan is not None:
+        return PythonCan(device_name, **kwargs)
```

Comment:
Move this condition one block higher, please. Otherwise, python_can would never be used on GNU/Linux.

Reply:
Any chance to have this one fixed?

Reply:
Done.